src/HPolyhedron.jl

```diff
@@ -601,7 +601,11 @@ function tovrep(P::HPoly{N};
     @assert isdefined(@__MODULE__, :Polyhedra) "the function `tovrep` needs " *
                                         "the package 'Polyhedra' to be loaded"
     P = polyhedron(P; backend=backend)
-    return VPolytope(P)
+    Q = VPolytope(P)
+    if Q isa VPolytope{N}
+        return Q
+    end
+    return VPolytope{N}(vertices_list(Q))
```

Comment:
The function that preserves the numeric type should be `polyhedron` with some backend, otherwise we should raise an error. Perhaps make it an assertion:

```julia
    @assert Q isa VPolytope{N} "the numeric type is not preserved by `polyhedron`"
    return Q
```

Reply:
At least `CDDLib` only supports `Float64` and `Rational` (so for instance no `Float32`). Then should we just remove those tests that work in `Float32`?

Reply:
I agree that we should only use `Float64` and `Rational`  with `CDDLib`.

Reply:
Okay. Should I remove the first commit as well (not needed anymore) or should we keep it?